perf/data.js

```diff
@@ -16,16 +16,17 @@ export function randomData(type, len, datumSize) {
  * @param {number} len
  */
 function randomDatum(type, len) {
-  if (type == 'string') {
-    return randomString(len);
-  } else if (type == 'object') {
-    return randomObject(len);
-  } else if (type == 'arraybuffer') {
-    return randomUint8Array(len).buffer;
-  } else if (type == 'blob') {
-    return randomBlob(len);
-  } else {
-    throw new Error('unsupported');
+  switch (type) {
```

Comment:
This responds to a review comment at https://github.com/rocicorp/replicache/commit/11426954d25c96077c49857c9abb7b3a5c07b6a7#r49230964, and I did it as all of these flagged `eqeqeq` previously.